authentication/managing-oauth-access-tokens.adoc

```diff
@@ -16,3 +16,6 @@ include::modules/oauth-view-details-tokens.adoc[leveloffset=+1]
 
 // Deleting user-owned OAuth access tokens
 include::modules/oauth-delete-tokens.adoc[leveloffset=+1]
+
+// Adding unauthenticated groups to ClusterRoleBindings
+include::modules/unauthenticated-users-cluster-role-binding.adoc[leveloffset=+1]
```

Comment:
When looking at the preview of the document, it feels surprising to bump into a call to action for adding unauthenticated groups to cluster role bindings.

Maybe something like: hey, in case that you run into issues or the behavior is odd, we removed the access for `system:unauthenticated` to the clusterrole `system:oauth-token-deleter`. Then we could have the module that lists the complete change or a specific module for `oauth-token-deleter` with a link to the release note that contains all changes.

Reply:
This module is reused for all the cluster roles so I don't want to call out a specific one in it. A specific module for `oauth-token-deleter` would need to wait until after GA at this point.

Reply:
Ok. I think for most of the docs we are adding it to, would benefit from having a dedicated intro / text tailored to the feature and then referencing the overall module. I would find it hard to understand which of the cluster roles are important or if all of them are and if it is a necessity to do that or not.